File: app/src/androidTest/java/bannerga/com/checkmytrain/NotificationJobTest.java
package bannerga.com.checkmytrain;

import android.content.Context;

import androidx.test.InstrumentationRegistry;

import org.junit.Assert;
import org.junit.Test;

import bannerga.com.checkmytrain.notification.NotificationJob;

public class NotificationJobTest {

    @Test
    public void scheduleJobWithHoursAndMinutes() {
        Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
        NotificationJob job = new NotificationJob();
        long offset = job.getOffsetInMillis(9, 39);
        int id = job.scheduleJob(context, "Muirend", "Glasgow Central", offset);
        Assert.assertTrue(id > 0);
    }
}

File: app/src/androidTest/java/bannerga/com/checkmytrain/NotificationJobTest.kt
package bannerga.com.checkmytrain

import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
import org.junit.Assert.assertTrue
import org.junit.Test


class NotificationJobTest {

    @Test
    fun scheduleJobWithHoursAndMinutes() {
        val job = MockNotificationJob(getInstrumentation().targetContext)
        job.scheduleJob("Muirend", "Glasgow Central", 0, 0)
        val id = job.jobId
        assertTrue(id > 0)
    }
}
